packages/vue/src/components/molecules/SfRadio/SfRadio.stories.js

```diff
@@ -99,7 +102,7 @@ const Template = (args, { argTypes }) => ({
   :disabled="disabled"
   :required="required"
   v-model="selected"
-  @input="input"
+  @change="change"
```

Comment:
why not leave both event?

Reply:
I didn't understand what you suggested before. So now it should good I think.